database/config.py

```diff
@@ -1,9 +1,15 @@
+import os
+import urlparse
 from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
-from sqlalchemy.ext.declarative import declarative_base
 
 
-engine = create_engine('mysql+mysqldb://root:root@localhost/ouija2', echo=True)
+DBURL = urlparse.urlparse(os.environ["DATABASE_URL"])
```

Comment:
@jmaher , hmmm so we have a little issue here. If we want to make sure people have a specific database configuration for them to test on locally, we must make a choose about which database we want to use as default.

If we wan to use **mysql**, the default configuration would be:
**database_url = 'mysql+mysqldb://root:root@localhost/ouija2'**

and if we set postgresql as default, it will become:

**database_url = 'postgresql+psycopg2://root:root@localhost/ouija2'**

NOTICE: I set database as ouija2 here is because we use ouija as our database name in alert manager server, we should switch to ouija as database name for avoiding the confusing, after the heroku server can work perfectly.

After one more thinking about this, I believe we could write a strategy to recognise the database running on the people's local environment and set database url dynamically. Like:
try:
""" If we get exception here, we believe it's running on local environment."""
os.environ["DATABASE_URL"]
except:
"""Figure out which database url we should init."""
"""But it may become pretty complex and it will make debug become harder if we have two different kinds of database."""